unitTests/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValueRendererTest.ts

```diff
@@ -0,0 +1,26 @@
+import { CategoryFacetValue } from '../../../../src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValue';
+import { CategoryFacetValueRenderer } from '../../../../src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValueRenderer';
+import { CategoryFacet, ICategoryFacetOptions } from '../../../../src/ui/CategoryFacet/CategoryFacet';
+import { CategoryFacetTestUtils } from '../CategoryFacetTestUtils';
+
+export function CategoryFacetValueRendererTest() {
+  describe('CategoryFacetValueRendererTest', () => {
+    let facetValue: CategoryFacetValue;
+    let facetValueRenderer: CategoryFacetValueRenderer;
+    let facet: CategoryFacet;
+
+    beforeEach(() => {
+      initializeComponent();
+    });
+
+    function initializeComponent(options?: ICategoryFacetOptions) {
```

Comment:
I find a more elegant pattern is to place the `options` object in the outer-most `describe` block scope (e.g. line 11). The object is reset in the `beforeEach`, and each tests changes properties as needed. When calling `initializeComponent`, there is no need to pass the object, since the function can retrieve the `describe` block instance.

All that being said, at the moment, it looks like the `options` arg is not being used, so this is more for a larger test suite where the options need to be individually changed.

Reply:
There are default options in the Utils, this is to pass an object to override/add new options, but I'll removve it for now